### ISSUE TITLE
[Due for payment 2026-02-23] Screen Reader: Many Pages: All chat messages are announced as 'chat message'

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77412]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe right to focus any chat and double-tap to activate it
3. Swipe right to focus any message in the chat and note how it is announced

Other Occurrences:
1. New Chat
2. Group Chat

## Expected Result:
Each chat message should be announced with the displayed details, including the sender name, time sent and message text, e.g.:
- 'Today at 10:17 AM, Concierge, Hey there'

## Actual Result:
All chat messages are announced simply as "chat message".
Screen reader users cannot determine who sent the message or what the message contains.

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Update each chat message container to expose the full, meaningful text that includes the sender, date and time, and the message content, e.g.:
- 'Today at 10:17 AM, Concierge, Hey there'
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2435" alt="Image" src="https://github.com/user-attachments/assets/bac2e367-1f2f-4ef2-bdec-460a8cd67064" />

https://github.com/user-attachments/assets/f738441a-1de8-43cd-95cb-0b4bb70bce75

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7009115
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
fixed https://github.com/Expensify/App/issues/79233
